src/definitions/modules/progress.js

```diff
@@ -333,10 +333,10 @@ $.fn.progress = function(parameters) {
           },
 
           percent: function(index) {
-            return module.percent[index || 0] || 0;
+            return module.percent && module.percent[index || 0] || 0;
```

Comment:
Note) Not only `module.value` but also `module.pecent` has same issue, so I fixed this too.

Reply:
👍